### PR TITLE
Documentation faqs

### PR DESCRIPTION
**Pull request by robnewman**
# Description
Add overview of using Quilt with R

# TODO
- [ ] [Changelog](CHANGELOG.md) entry

### CODE REVIEW

File: docs/FAQ.md [hunk 1]
```diff
@@ -64,3 +64,31 @@ a local machine or foreign region)—I/O is much faster.
...
+
+### Using the Quilt CLI API with R
+You can script the Quilt CLI directly from your shell environment and chain it
+with your R scripts to create a unified workflow:
+
+```bash
+quilt3 install
+[Run R commands or scripts]
+quilt3 push
+```
```
Comment (drernie) [inline docs/FAQ.md:90]: I find it hard to understand follow what  is happening without context. Perhaps:
```
$ quilt3 install # download package
[Run R commands or scripts] # modify or add files
$ quilt3 push # upload package
```
Unless there's a different convention that works better?

**Review by drernie — APPROVED** [review]

File: docs/FAQ.md (hunk 1 above)
Comment (robnewman) [reply docs/FAQ.md:90]: I'm not sure you can isolate this to just a CLI (i.e. the $ CLI prefix). You can have a bash script that does all this in one executable

Comment (robnewman) [reply docs/FAQ.md:90]: I followed the convention used earlier in the page

Comment (robnewman) [reply docs/FAQ.md:90]: I've added more context and resubmitted

**Review by drernie — DISMISSED** [review]

**Comment by robnewman** [comment]
@drernie Need another review please

**Review by drernie — DISMISSED** [review]

**Comment by drernie** [comment]
FYI, in an ideal world I would want that bash script to be an actual runnable code example, so we could test that it was valid.  However, that requires adding write-capable credentials to our CI environment, which is not worth burdening this PR with. I'll create a new Asana for that...

**Comment by robnewman** [comment]
@drernie Unfortunately adding that failed (perhaps because I didn't add it to the first line) so I added just a `<!--pytest.mark.skip-->` for the code block in question. Hopefully that resolves the issue

**Review by drernie — APPROVED** [review]
Ah, sorry. Looks good now.

**Pull request merged**